app/models/payment.rb

```diff
@@ -142,4 +143,9 @@ def check_amount
       throw :abort
     end
   end
+
+  def send_webhook
+    webhook = WebhookPayment.new(self)
+    WebhookJob.perform_later(webhook.endpoint, webhook.payload)
```

Comment:
this is hard to follow. how do the webhook.endpoint and webhook.payload values get set?

Reply:
also doesn't seem right to call the job here. the webhook should know if it needs to use queuing, not the payment.

Reply:
That makes sense about calling another method that then does the perform_later. Re: webhook.endpoint and webhook.payload, that's a good point... OH, actually moving the actual ActiveJob call into the Webhook class will solve that other issue too. The problem was that perform_later only takes certain basic variable types - not classes.